src/accelerate/commands/launch.py

```diff
@@ -343,14 +343,14 @@ def launch_command_parser(subparsers=None):
     )
     deepspeed_args.add_argument(
         "--zero3_init_flag",
-        default=None,
+        default="true",
         type=str,
         help="Decides Whether (true|false) to enable `deepspeed.zero.Init` for constructing massive models. "
         "Only applicable with DeepSpeed ZeRO Stage-3.",
     )
     deepspeed_args.add_argument(
         "--zero3_save_16bit_model",
-        default=None,
+        default="false",
         type=str,
         help="Decides Whether (true|false) to save 16-bit model weights when using ZeRO Stage-3. "
         "Only applicable with DeepSpeed ZeRO Stage-3.",
```

Comment:
may be add to `help` that `ACCELERATE_DEEPSPEED_ZERO3_SAVE_16BIT_MODEL` env var can override it?

Reply:
Same explanation as above